libs/model/src/config.ts

```diff
@@ -195,6 +197,7 @@ export const config = configure(
       MIN_USER_ETH: z.number(),
       MAX_USER_POSTS_PER_CONTEST: z.number().int(),
       FLAG_FARCASTER_CONTEST: z.boolean().nullish(),
+      NEYNAR_BOT_UUID: z.string().nullish(),
```

Comment:
Can we add a `refine` step for these variables and tie them to some flag? That way when the flag is flipped we know the variable _must_ be present?